### ISSUE TITLE
All data and config in separate drive

### ISSUE THREAD

**Issue by toshke** [toshke/big-blue-button-cloudformation-cfhl#8]

**Comment by ghendo** [comment]
wouldnt this be very complicated with redis/mongo/postgres etc etc etc all needing to be pointed somewhere.  This would be great though for a better cloud setup with managed services communicating with fixed code

**Comment by toshke** [comment]
Yeah, this would introduce a new level of complexity to the project, i left it as an braindump of idea w/out any specific implementation plan. For time being, AMI backups should suffice - I would expect that default setup would work for most of the users with appropriate disk / instance size parameters.
